Acme/Certificate/Storage/CertificateStorageFactory.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * This file is part of the ACME PHP library.
+ *
+ * (c) Titouan Galopin <galopintitouan@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace AcmePhp\Bundle\Acme\Certificate\Storage;
+
+use Symfony\Component\Filesystem\Filesystem;
+
+/**
+ * Create CertificateStorage.
+ *
+ * @author Jérémy Derussé <jeremy@derusse.com>
+ */
+class CertificateStorageFactory
```

Comment:
CertificateStorage has a dependency on the current domain.

I'm not sur it's neither a good idea to pass a parameter `$domain` in each call